tests/ordering/tests.py

```diff
@@ -190,6 +197,12 @@ def test_order_by_nulls_first(self):
             ),
             [self.a1, self.a2, self.a4, self.a3],
         )
+        self.assertQuerySetEqualReversible(
+            Article.objects.annotate(upper_name=Upper("author__name")).order_by(
+                F("upper_name").desc(nulls_first=True), "headline"
+            ),
+            [self.a1, self.a2, self.a4, self.a3],
+        )
```

Comment:
I guess these two tests could be added in a separate commit that refs ticket-33308 as support for order by position on backends that don't support native `NULLS (FIRST|LAST)` related to changes required to support served side bindings for Pyscopg 3.

Reply:
Agreed, see #16572.